cogs/moderation.py

```diff
@@ -293,6 +298,96 @@ async def clear(self, ctx: Context, amount: int) -> None:
+
+        for embed in reversed(messages):
+            await target.send(embed=embed)
+
+    @command()
+    @has_permissions(manage_messages=True)
+    async def dm(self, ctx: Context, members: Greedy[Member], *, message: str) -> None:
+        """Dm a List of Specified User from Your Guild."""
+        embed = Embed(
+            title="Notice!",
+            description=message,
+            color=Color.dark_magenta()
+        )
+        embed.set_footer(text=f"From {ctx.guild.name}", icon_url=ctx.guild.icon_url)
+
+        for member in members:
+            await member.send(embed=embed)
+
+    @command()
+    @has_permissions(administrator=True)
+    async def dmall(self, ctx: Context, *, message: str) -> None:
+        """Dm all Users from Your Guild."""
+        embed = Embed(
+            title="Notice!",
+            description=message,
+            color=Color.dark_magenta()
+        )
+        embed.set_footer(text=f"From {ctx.guild.name}", icon_url=ctx.guild.icon_url)
+
+        for member in ctx.guild.members:
+            with suppress(Forbidden):
+                await member.send(embed=embed)
+
+    @command()
+    @has_permissions(manage_roles=True)
+    async def lock(self, ctx, channels: Greedy[TextChannel] = None, reason: str = 'Not Specified') -> None:
+        """Disable @everyone's permission to send message on given channel or current channel if not specified."""
+        if channels is None:
+            channels = [ctx.channel]
+        for c in channels:
+            await c.set_permissions(c.guild.default_role, send_messages=False,
+                                    reason=f"Reason: {reason} | Requested by {ctx.author}.")
+            await c.send("🔒 Locked down this channel.")
+        if not channels == [ctx.channel]:
+            await ctx.send(f"Locked down {len(channels)} channel{'s' if len(channels) > 1 else ''}.")
+
+    @command()
+    @has_permissions(manage_roles=True)
+    async def unlock(self, ctx, channels: Greedy[TextChannel] = None, reason: str = 'Not specified') -> None:
+        """Reset @everyone's permission to send message on given channel or current channel if not specified."""
+        if channels is None:
+            channels = [ctx.channel]
+        for c in channels:
+            await c.set_permissions(c.guild.default_role, send_messages=None,
+                                    reason=f"Reason: {reason} | Requested by {ctx.author}.")
+        await ctx.send(f"Unlocked {len(channels)} channel{'s' if len(channels) > 1 else ''}.")
+
+    @command()
+    @has_permissions(manage_channels=True)
+    async def slowmode(self, ctx, channels: Greedy[TextChannel] = None, seconds: int = 10, reason: str = "Not specified") -> None:
+        """Set channel's slowmode delay, default = 10s."""
+        if not (0 <= seconds <= 21600):
+            await ctx.send(":x: The duration is extreme or very low.")
+            return
+
+        if channels is None:
+            channels = ctx.channel
```

Comment:
The variable name is `channels` it should be referencing to multiple channels, so it should be a list
```suggestion
            channels = [ctx.channel]
```